### ISSUE TITLE
False postive: arguments annotated as metaclass instances are thought to be unsubclassable.

### ISSUE THREAD

**Issue by patrick-kidger** [microsoft/pyright#4737]
```python
class Meta(type):
    pass


def subclasses_meta(cls: Meta):
    class SubCls(cls):
        ...
```
produces `error: Expected type expression but received "Meta" (reportGeneralTypeIssues)`.

However `cls: Meta` indices that `cls` is an instance of a metaclass, which implies that it is a class, which implies it is subclassable.

Python 3.8.0.
Pyright version 1.1.296 run from the command line.

**Comment by erictraut** [comment]
This will be addressed in the next release.

Incidentally, mypy contains the same bug.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.297, which I just published. It will also be included in a future release of pylance.
